lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -287,6 +276,24 @@ defmodule Trento.Domain.SapSystem do
     |> Multi.execute(&maybe_emit_sap_system_tombstoned_event/1)
   end
 
+  def execute(
```

Comment:
The same as others.
I think we should only leave the database and application registration `execute`s on top of this.
For the application is not really possible, as we cascade multiple events together.
But we could simply leave those 2 functions on top